### PR TITLE
Query and  table update

### PR DESCRIPTION
**Pull request by richie1988**
- ###  query and update animals table
- Added a column species of type string to My animal's table. Modified the schema.sql file.
- ### **I Inserted the following data:**
- Animal: His name is Charmander. He was born on Feb 8th, 2020, and currently weighs -11kg. He is not neutered and he has never tried to escape.
- Animal: Her name is Plantmon. She was born on Nov 15th, 2021, and currently weighs -5.7kg. She is neutered and she has tried to escape 2 times.
- Animal: His name is Squirtle. He was born on Apr 2nd, 1993, and currently weighs -12.13kg. He was not neutered and he has tried to escape 3 times.
- Animal: His name is Angemon. He was born on Jun 12th, 2005, and currently weighs -45kg. He is neutered and he has tried to escape once.
- Animal: His name is Boarmon. He was born on Jun 7th, 2005, and currently weighs 20.4kg. He is neutered and he has tried to escape 7 times.
- Animal: Her name is Blossom. She was born on Oct 13th, 1998, and currently weighs 17kg. She is neutered and she has tried to escape 3 times.
- Animal: His name is Ditto. He was born on May 14th, 2022, and currently weighs 22kg. He is neutered and he has tried to escape 4 times.
- Remember to add these insertions to your data.sql file.
- Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
- Take a screenshot of the results of your actions.
- Inside a transaction:
- I Updated the animals table by setting the species column to digimon for all animals that have a name ending in mon.
- I Updated the animals table by setting the species column to pokemon for all animals that don't have species already set.
- I Verified that All changes were made.
- Commit the transaction.
- I Verified that changes persist after the commit.
- After the rollback, I verified all records in the animal's table still exist. 

- ### Inside a transaction:
- I updated the animal's table by setting the species column to unspecified. i did Verify that changes were made. 
 - roll back the changes and I verified that the species columns went back to the state before the transaction.
- Inside a transaction:
- Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
- Update the animals table by setting the species column to Pokemon for all animals that don't have species already set.
- Verify that changes were made.
- Commit the transaction.
- Verify that changes persist after commit.
- Take a screenshot of the results of your actions.
- Now, take a deep breath and... Inside a transaction delete all records in the animal's table, then roll back the transaction.
- After the rollback verify if all records in the animal's table still exist. After that, you can start breathing as usual ;)
### Screen shot for above transaction 
00query
1query
3query
-3query

- I Deleted all animals born after Jan 1st, 2022.
- I Create a savepoint for the transaction.
- I UpdateD all animals' weight to be their weight multiplied by -1.
- 
- Then Update all animals' weights that are negative to be their weight multiplied by -1.
- I Committed the  transaction
- How many animals are there?
- How many animals have never tried to escape?
- What is the average weight of animals?
- Who escapes the most, neutered or not neutered animals?
- What is the minimum and maximum weight of each type of animal?
- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
- ### SCREENSHOTS BELOW
- 
Code_5uVOYcb3fQ
Code_FudtV4vRJ7
Code_i18Xv0pfJW
Code_8xWDmcAfhQ
Code_M3rXYcAuhs
Code_dDLCwmek85
Code_OXcgJa8jj5
Code_FrgXMayu5d
Code_FudtV4vRJ7
Code_i18Xv0pfJW
Code_M3rXYcAuhs
Code_OXcgJa8jj5
Code_tTo1XdUwvP
Code_UYbtu7zoDV
Code_ZVI5zgu4wZ

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -8,3 +8,17 @@ SELECT * FROM animals WHERE neutered = true;
...
+SELECT AVG(weight_kg) AS average_weight FROM animals;
+SELECT neutered, MAX(escape_attempts) AS max_escape_attempts
+FROM animals
+GROUP BY neutered;
+SELECT species, MIN(weight_kg) AS min_weight, MAX(weight_kg) AS max_weight
+FROM animals
+GROUP BY species;
+SELECT species, AVG(escape_attempts) AS avg_escape_attempts
+FROM animals
+WHERE YEAR(date_of_birth) BETWEEN 1990 AND 2000
```
Comment (mikethreels) [inline queries.sql]: - [ ] it seems that you have not pushed the below part of the requirements to the repository. please make sure to include these.

image

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Richard, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- good job adding the data to the table

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (richie1988) [reply queries.sql]: Thank you your highlight i thought i did add maybe when i was making changes i made a mistake thank you

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @richie1988 ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Implemented requested changes ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**